conda/__init__.py

```diff
@@ -16,6 +16,24 @@
     reload(sys)  # NOQA
     sys.setdefaultencoding('utf-8')
 
+# hack: preload openssl and libarchive on windows, so we can be sure they're loaded correctly
+if sys.platform == "win32":
+    from ctypes import cdll
+    path_backup = os.environ['PATH']
+    # temporarily change PATH, such that implicit loads (deps of our libs) can be picked up
+    os.environ['PATH'] = ';'.join((join(sys.prefix, "Library", "bin"), path_backup))
+    # we have openssl 1.0.2 and 1.1.1 filenames here.  If they're not found, it doesn't matter.
+    libraries = ("archive.dll",
+                 "libcrypto-1_1-x64.dll", "libssl-1_1-x64.dll",
+                 "libcrypto-1_1.dll", "libssl-1_1.dll",   # x86
+                 "libeay32.dll", "ssleay32.dll"
+    )
+    for library in libraries:
+        lib_path = join(sys.prefix, "Library", "bin", library)
+        if os.path.exists(lib_path):
+            cdll.LoadLibrary(lib_path)
```

Comment:
Should we use a try/except here in case the `LoadLibrary` fails?  If these files were corrupt conda would be in a pickle so failing early might be reasonable behavior.

Reply:
Good point. I think if these files are hosed, conda is not going to be doing much one way or another. Maybe this is a good place for us to provide a better error message for library load failures than people would otherwise see down the line. Libarchive has an especially cryptic message. Easy enough to discern from the traceback, but yeah, who reads those?

Reply:
I removed this.  I hoped it might fix win-32, but it didn't.  It's a nasty hack, and we shouldn't have it if it isn't helping.